direnv.el

```diff
@@ -66,17 +66,19 @@ usually results in coloured output."
 (defcustom direnv-non-file-modes '(eshell-mode dired-mode)
   "List of modes where direnv will update even if the buffer has no file.
 
-In these modes, direnv will use `default-directory' instead of
-`(file-name-directory (buffer-file-name (current-buffer)))'."
+In these modes, or modes derived from them, direnv will use
+  `default-directory'
+instead of
+  `(file-name-directory (buffer-file-name (current-buffer)))'."
   :group 'direnv
   :type '(repeat (symbol :tag "Major mode")))
 
 (defun direnv--directory ()
   "Return the relevant directory for the current buffer, or nil."
   (let* ((buffer (or (buffer-base-buffer) (current-buffer)))
-         (file-name (buffer-file-name buffer)))
+	 (file-name (buffer-file-name buffer)))
```

Comment:
hmmm github shows a change on this line. is there some problematic whitespace here perhaps?

Reply:
Ugh, I have a tab.